app/services/user_validation_service.rb

```diff
@@ -87,7 +87,7 @@ def missing_user_features(db_user)
   end
 
   def user_is_super_admin?
```

Comment:
@kbrock I'm thinking this can be removed and the couple places that call it replaced with the new helper version.